### DISCUSSION TITLE
How to create public link on Windows 11?

### DISCUSSION THREAD

**Discussion by RauchenwaldC** [Woolverine94/biniou#33] [Q&A] [answered]
`To create a public link, set `share=True` in `launch()`.`

How/where to do that on Windows 11 when I start biniou by double-clicking on the webui.cmd file?

**Comment by Woolverine94** [comment, marked as answer]
Hello @RauchenwaldC,

The message you are referring to is a standard gradio text output, and has nothing to do with biniou. 

Thus, you can easily opening a public access by : 

  - Editing file biniou\.ini\auth.cfg and defining a new user:password couple, instead of the standard biniou:biniou
  - (re)start biniou and go to the "Global settings" tab
  - Use your brand new defined credentials to login
  - Go to the "WebUI Control" Tab 
  - In the "Backend settings" box, check the "Activate authentication" checkbox and the "share online" checkbox
  - Click the "Save custom defaults settings" a the bottom of the page
  - (re)start biniou
  - Public link will be displayed in the terminal output
